google-cloud-pubsub/lib/google/cloud/pubsub/received_message.rb

```diff
@@ -179,9 +181,9 @@ def published_at
         #   # Shut down the subscriber when ready to stop receiving messages.
         #   subscriber.stop!
         #
-        def acknowledge!
+        def acknowledge! &block
```

Comment:
This is my lack of ruby skills: Is this change impacting existing usage of acknowledge? Same question for the rest of the changes in this class.

I see we have changes in the usage comments. Do we need to provide example for previous usage?

Reply:
This is not a usage change. The `&` "argument" is just a syntax for capturing any passed block as an argument.